drivers/overlay/mount.go

```diff
@@ -141,14 +141,27 @@ func mountOverlayFromMain() {
 	// the new value for the list of lowers, because it's shorter.
 	if lowerv != "" {
 		lowers := strings.Split(lowerv, ":")
+		var newLowers []string
+		dataOnly := false
 		for i := range lowers {
```

Comment:
(Absolutely non-blocking: `for _, lowerPath := range lowers` (or even `range strings.Split…)` might cut down on the visual noise a bit.)

Reply:
fixed